### PR TITLE
Add spec for custom matchers and some extra chainables

### PR DESCRIPTION
**Pull request by Crisfole**
Update RSpec matchers with extra chains and add some tests (but exclude rspec.rb)

### CODE REVIEW

**Comment by Crisfole** [comment]
@egreer @ncallaway at least one of you should take a look here

**Comment by wkirby** [comment]
This addresses #11

File: README.md [hunk 1]
```diff
@@ -248,22 +248,55 @@ RSpec.describe RSpecCommand do
...
+      it { is_expected.to be_failed_result.with_message("message") }
+      it { is_expected.to be_failed_result.with_value("value") }
     end
   end
 end
 ```
 
 ### Minitest
 
+To use with Minitest, updaet your 'test_helper' file to include:
```
Comment (egreer) [inline README.md]: `update`

File: spec/lib/custom_matcher_spec.rb [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+      it 'with message' do
+        expect(WhateverCommand.call(message: 'Hire Apsis Labs!', succeed: true)).to be_success_result.with_message('Hire Apsis Labs!')
+      end
+
+      it 'with status' do
+        expect(WhateverCommand.call(status: :apsis_rocks, succeed: true)).to be_success_result.with_status(:apsis_rocks)
+      end
+    end
+
+    context 'failure', skip: 'When we figure out how to test the failure messages reenable' do
```
Comment (egreer) [inline spec/lib/custom_matcher_spec.rb]: Why can't we test the failures?

**Comment by egreer** [comment]
Seems fine outside of the small comments.

**Pull request merged**